lib/src/screens/session/widgets/arrow_back_screen.dart

```diff
@@ -10,12 +10,15 @@ import '../../home/home_screen.dart';
 
 class ArrowBack extends StatefulWidget {
   final bool success;
-  final int amountIssued;
+  final bool isIssuanceSession;
+  final bool isSignatureSession;
```

Comment:
Have you thought to use an enum? Like:

`enum SessionType { issuance, signature }`

This way it can never happen both isIssuanceSession and isSignatureSession are true at the same time (although the impact is very limited of coarse).